pkg/streamingpromql/engine_test.go

```diff
@@ -58,132 +58,142 @@ func TestUnsupportedPromQLFeatures(t *testing.T) {
 
 	for expression, expectedError := range unsupportedExpressions {
 		t.Run(expression, func(t *testing.T) {
-			requireQueryIsUnsupported(t, featureToggles, expression, expectedError)
+			requireQueryIsUnsupported(t, mqeOpts, expression, expectedError)
 		})
 	}
 }
 
-func TestUnsupportedPromQLFeaturesWithFeatureToggles(t *testing.T) {
+func TestUnsupportedPromQLFeaturesWithMQEOpts(t *testing.T) {
```

Comment:
[nit] `TestUnsupportedPromQLFeaturesWithFeatureToggles` seems like a clearer name to me.